mungegithub/mungers/publish_scripts/publish_driver.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
This shows how i used the other scripts to generate the changes. This script will be removed before merging.